types/index.d.ts

```diff
@@ -2072,6 +2072,19 @@ declare namespace math {
       defaultValue?: number | string
     ): T
 
+    /**
+     * Return a Rotation Matrix for a given angle in radians
+     * @param {number | BigNumber | Complex | Unit} theta    Rotation angle
+     * @param {Array | Matrix} [v]                           Rotation axis
+     * @param {string} [format]                              Result Matrix storage format. Default value: 'dense'.
+     * @return {Matrix}                                      Rotation Matrix
+     */
+    rotationMatrix<T extends Matrix>(
```

Comment:
I think `T extends Matrix` should be replaced with `T extends MathCollection`: it can return both a Matrix or an Array